modules/orca/output.tf

```diff
@@ -18,3 +18,24 @@ output "orca_lambda_copy_to_glacier_arn" {
   value = module.orca_lambdas.copy_to_glacier_arn
 }
 
+
+
+
+
+output "orca_sqs_get_message_queue_arn" {
+  value = module.orca_sqs.get_message_queue_arn
+}
+
+output "orca_sqs_get_message_queue_id" {
```

Comment:
Let's specify the purpose for this queue a bit more... 'copy_files_to_archive_queue'?

Reply:
yes I should write a "Description" parameter here as well that describes what the queue does